phoenix-core-client/src/main/java/org/apache/phoenix/filter/SkipScanFilter.java

```diff
@@ -618,7 +632,8 @@ public void readFields(DataInput in) throws IOException {
                 orClause.add(range);
             }
         }
-        this.init(slots, slotSpan, schema, includeMultipleVersions);
+        boolean isPointLookup = in.readBoolean();
```

Comment:
For backward compatibility we need to handle EOFException because the older clients won't be sending the boolean field

Reply:
Good point! I will do that.